README.md

```diff
@@ -223,5 +224,21 @@ And, in your `main.rs` or `lib.rs`:
 #![cfg_attr(feature="clippy", plugin(clippy))]
 ```
 
+## Configuration
+Some lints can be configured in a `Clippy.toml` file. It contains basic `variable = value` mapping eg.
+
+```toml
+blacklisted-names = ["toto", "tata", "titi"]
+cyclomatic-complexity-threshold = 30
+```
+
+See the wiki for more information about which lints can be configured and the
+meaning of the variables.
+
+You can also specify the path to the configuration file with:
+```rust
+#![plugin(clippy(conf_file="path/to/clippy's/configuration"))]
```

Comment:
I use that in the tests, I’m not sure it’s useful enough to be worth mentioning in the README tough. Maybe the wiki?